q2_alignment/_mafft.py

```diff
@@ -91,9 +92,13 @@ def _mafft(sequences_fp, alignment_fp, n_threads, parttree, addfragments):
     if parttree:
         cmd += ['--parttree']
 
+    if keeplength is not None:
```

Comment:
```suggestion
    if keeplength\:
```

Reply:
I just realized this myself! 🤦‍♂️ 

I guess I am not as quite as recovered as I thought. 😬